### ISSUE TITLE
Able to get vault cookies after lvl 25

### ISSUE THREAD

**Issue by ChromoSphere-Me** [Iskallia/Vault-public#752]
**Describe the bug**
It is possible to get trader cores that sell you vault cookies even after lvl 25. thus becoming useless trades.

**To Reproduce**
well its luck based if you get it. so the only real way would be get over lvl 25 and pickup trader cores till you get one that trades for vault cookies.

**Expected behavior**
Well after lvl 25 you shouldn't be getting vault cookies as trades they are useless at that point.

**Screenshots or Crash Report**
![2022-01-28_15 42 10](https://user-images.githubusercontent.com/61015811/151566561-362893e0-9508-4b76-bc31-dc988eb5e7f8.png)

**Info (please complete the following information):**
- World Type: Server

- Vault hunters modpack version: 1.12.4

**Comment by Ithkamakazi** [comment]
that was a fresh trader core that wasn't handled by any other player before yourself at vault lvl 25+ or any other player above lvl 25?

**Comment by ChromoSphere-Me** [comment]
yes picked it up in the vault and came like that out of my dank. i was at lvl 44 at that time
